packages/code-du-travail-frontend/src/layout/css/Root.js

```diff
@@ -28,9 +22,14 @@ export default createGlobalStyle`
     text-rendering: optimizeLegibility;
     background: ${colors.darkBackground};
     border-top: 5px solid ${colors.blue};
+    @media print {
+      font-size: 10pt;
```

Comment:
should we move this in the `print.css` or remove print.css and move rules all in components ?

Reply:
seems you move rules to components ! 👍